test/optimal_filter_validation.jl

```diff
@@ -98,7 +98,7 @@ Sobs = 0.01
 #### Example Specification of Covariance Function
 function R(x::Float64, y::Float64, th)
 
-  Cov = (th.s^2)*exp(-(abs(x)+abs(y))/(2.0*th.l))
+  Cov = (th.s^2)*exp(-hypot(x, y) / th.l)
```

Comment:
These two expressions don't look equal, was this an intentional correction?

Reply:
See discussion above https://github.com/Team-RADDISH/ParticleDA.jl/pull/216#discussion_r793748017 :wink:

Reply:
Ah! I didn't notice the resolved discussion